src/components/IncomeStatement.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import Back from './Back';
+import styles from '../styles/Details.module.scss';
+import Filter from './Filter';
+
+const IncomeStatement = () => {
```

Comment:
- [ ] I see you have search functionality in all the components. You are free to implement this search in all components as your own added preference however you should have a search feature in the Home component as stated in the requirements.